server/url_prefixer.go

```diff
@@ -80,9 +80,9 @@ func (up *URLPrefixer) ServeHTTP(rw http.ResponseWriter, r *http.Request) {
 	// extract the flusher for flushing chunks
 	flusher, ok := rw.(http.Flusher)
 	if !ok {
-		up.Logger.
-			WithField("component", "prefixer").
-			Fatal("Expected http.ResponseWriter to be an http.Flusher, but wasn't")
+		msg := "Expected http.ResponseWriter to be an http.Flusher, but wasn't"
```

Comment:
Yeah, it's definitely a precondition for chunk encoding and prefixing in general. We could continue safely by forwarding to the embedded writer as long as there's no prefix set by the user. Otherwise we should probably fail telling the user to unset the prefix flag since the page will be broken without assets.